### DISCUSSION TITLE
Word wrap cells

### DISCUSSION THREAD

**Discussion by aborruso** [saulpw/visidata#1080] [Q&A] [answered]
Hi,
a very stupid question. I remember that there is a way to set row width, running a command that apply word wrap to column cell content. Am I wrong?

If no, I have searched in [documentation](https://www.visidata.org/docs/man/)  but I do not find it.

Thank you

**Comment by geekscrapy** [comment, marked as answer]
If you are looking for word wrapping, use "`v`", but if you want to set the column width its "`z_`"

**Reply by LanHikari22** [reply]
Hi. Maybe this should be added to the [column documentation](https://www.visidata.org/docs/columns/) and/or [cheat sheet](https://jsvine.github.io/visidata-cheat-sheet/en/) ?

Also it appears even after using v, there may still be some characters not showing with ... 
Especially if you press enter on that row and then press v

It also can help to do z+Enter and then v in some cases where the text is too large.
